code/controllers/configuration/entries/comms.dm

```diff
@@ -24,3 +24,9 @@
 
 /datum/config_entry/string/cross_comms_network
 	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/string/hell
+	protection = CONFIG_ENTRY_LOCKED // no trying to hotswap the config to shoot people to citadel, that's too cruel even for us
+
+/datum/config_entry/string/the_abyss
+	protection = CONFIG_ENTRY_LOCKED
```

Comment:
These must take a name of a server in the cross-comms list, and allow editing (since editing that list isn't allowed). There is no reason to protect these configs when other options exist.

>={

Reply:
yeah I only put those there so I could make the citadel joke, I've removed the lock and will probably link it to the cross-comm list if this actually works